VRDR/IJEMortality.cs

```diff
@@ -777,8 +777,8 @@ private void Set_MappingIJEToFHIR(Dictionary<string, string> mapping, string ije
         /// <summary>NCHS ICD10 to actual ICD10 </summary>
         private string NCHSICD10toActualICD10(string nchsicd10code)
         {
-            Regex ICD10rgx = new Regex(@"^[A-Z]\d{2}(\.\d){0,1}$");
-            Regex NCHSICD10rgx = new Regex(@"^[A-Z]\d{2,3}$");
+            Regex ICD10rgx = new Regex(@"^[A-Z]\d{2}(\.\d){0,1}$");  // ICD10 codes only have one character after the decimal
+            Regex NCHSICD10rgx = new Regex(@"^[A-Z]\d{2,4}$");       // NCHS tacks on an extra character to some ICD10 codes, e.g., K7210 (K27.10)
             string code;
             nchsicd10code = nchsicd10code.Trim();
             if (ICD10rgx.IsMatch(nchsicd10code))
```

Comment:
Does this case (code is already an ICD10 code so don't change it) basically come into play for three character codes?

Reply:
Yes.    A 3 Char NCHS code is a valid ICD10 code.